recipes/nest-desktop/meta.yaml

```
@@ -0,0 +1,45 @@
+{% set name = "nest-desktop" %}
+{% set version = "3.1.2" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/nest-desktop-{{ version }}.tar.gz
+  sha256: 49051aeefc8be189ae7dfb64ad983b7a0a5636765a79febd3347194b57ba61ba
+
+build:
+  number: 0
+  script: {{ PYTHON }} -m pip install . -vv
+  noarch: python
+
+requirements:
+  host:
+    - pip
+    - python >=3.6
+  run:
+    - python >=3.6
+
+test:
+  imports:
+    - nest_desktop
+  commands:
+    - pip check
+    - nest-desktop start  # [unix]
```

when it should run as daemon:

```suggestion
    - nest-desktop start -d  # [unix]
```